ladybug/designday.py

```diff
@@ -31,9 +31,27 @@
 class DDY(object):
     """A DDY object containing all of the data of a .ddy file.
 
-    Properties:
+    Args:
         location: A Ladybug location object
         design_days: A list of the design days in the ddy file.
+
+    Properties:
+        * hourly_barometric_pressure
+        * hourly_datetimes
+        * hourly_dew_point
+        * hourly_dry_bulb
+        * hourly_horizontal_infrared
+        * hourly_relative_humidity
+        * hourly_sky_cover
+        * hourly_solar_radiation
+        * hourly_wind_direction
+        * hourly_wind_speed
+        * humidity_condition
+        * isDesignDay
+        * location
+        * name
+        * sky_condition
+        * wind_condition
```

Comment:
I just realized that these properties are in the incorrect location. They shouldn't be non the `DDY` class and they should instead be on the `DesignDay` class.  The only properties that should appear here on the `DDY` object are:
```
* file_path
* location
* design_days
```